controllers/tools.js

```diff
@@ -179,18 +179,21 @@ router.get('/search' , function (req, res, next) {
     const itemsPerPage = 10;
```

Comment:
If I am on the search route an create a new tool it leads to an error (Not found) ... don't use current URL location when POST the modal for creation of a new tool.